src/test/resources/feature/3-destinatario/personaFisica/downloadFile/downloadAttestazioneOpponibilePresaInCarico.feature

```diff
@@ -72,10 +76,14 @@ Feature: persona fisica scarica attestazioni all'interno di una notifica
     And Logout da portale persona fisica
 
 
-      @@TA_PFVerificaEnteNelDettaglioNotifica
+  @@TA_PFVerificaEnteNelDettaglioNotifica
   Scenario: PN-10427 - Persona fisica verifica concatenazione nome ente Radice e ente Figlio in dettaglio notifica
-    Given Login Page mittente "mittente_1" viene visualizzata
-    When Login con mittente "mittente_1"
+    Given Login Page mittente viene visualizzata
```

Comment:
there is no need to enter the URL since we are talking about the sender, if you want to enter the URL as a parameter make the step generic for the 3 users

Reply:
I entered url, because in the same step with yaml file it uses the url. So i just transferred url from yaml to gherkin as a parameter